Makefile

```diff
@@ -71,11 +75,19 @@ docker: certs build-in-docker pause-container-release cni-plugins .out-stamp
 	@docker build -f scripts/dockerfiles/Dockerfile.release -t "amazon/amazon-ecs-agent:make" .
 	@echo "Built Docker image \"amazon/amazon-ecs-agent:make\""
 
+ifeq (${TARGET_OS},windows)
+    BUILD="cleanbuild-${TARGET_OS}"
+    GO_VERSION=${GO_VERSION_WINDOWS}
```

Comment:
when do you use `GO_VERSION` vs `O_VERSION_LINUX` vs `GO_VERSION_WINDOWS` throughout the make files? 

I guess we can just use `GO_VERSION` everywhere?